crates/oxc_linter/src/snapshots/consistent_type_definitions.snap

```diff
@@ -0,0 +1,168 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: consistent_type_definitions
+---
+  ⚠ typescript-eslint(consistent-type-definitions):
+   ╭─[consistent_type_definitions.tsx:1:10]
+ 1 │ type T = { x: number; };
+   ·          ───────┬──────
+   ·                 ╰── Use an `interface` instead of a `type`
+   ╰────
+  help: Use an `interface` instead of a `type`
+
+  ⚠ typescript-eslint(consistent-type-definitions):
+   ╭─[consistent_type_definitions.tsx:1:8]
+ 1 │ type T={ x: number; };
+   ·        ───────┬──────
+   ·               ╰── Use an `interface` instead of a `type`
+   ╰────
+  help: Use an `interface` instead of a `type`
+
+  ⚠ typescript-eslint(consistent-type-definitions):
+   ╭─[consistent_type_definitions.tsx:1:33]
+ 1 │ type T=                         { x: number; };
+   ·                                 ───────┬──────
+   ·                                        ╰── Use an `interface` instead of a `type`
+   ╰────
+  help: Use an `interface` instead of a `type`
```

Comment:
It points to `typeAnnotation`, which may not helpful for this error.

Reply:
makes sense, will take a look how to fix it